### PR TITLE
Add a new feature: Hyper link menu

### PR DESCRIPTION
**Pull request by ibrahim-s**
Hi @cary-rowen 
Add a new feature named hyper link menu.
This new feature helps us to open a hyper link with the browser we want.
- Stand on a link in a browser or in a message body
- Press Alt+/ (the default gesture of the addon)
- A virtual menu will pop up, giving you the option to open the link with several browsers on your machine.
- You can press the escape key to dismiss the menu, or enter on any of the browsers name, and the link will open in that browser.
Any suggestions or modifications is welcomed.

### CODE REVIEW

File: addon/globalPlugins/openLinkWith/__init__.py [hunk 1]
```diff
@@ -117,6 +188,51 @@ def script_displayLinksInLastSpokenText(self, gesture):
 			DIALOG= MyDialog(gui.mainFrame, list_, browsers)
 			DIALOG.postInit()
 
+	@script(
+		# Translators: Message to be displayed in input help mode.
+		description= _("If standing on a link, display a menu "
+		"to give you the option to open the link with other browsers on  your machine."
+		),
+		gesture= "kb:alt+/"
```
Comment (cary-rowen) [inline addon/globalPlugins/openLinkWith/__init__.py:196]: It’s really a good feature, the Alt+/ gesture is convenient enough, but this gesture does not include the NVDA key, will it conflict with other software? Although I can't think of an exact example right now.

Comment (ibrahim-s) [reply addon/globalPlugins/openLinkWith/__init__.py:196]: > It’s really a good feature, the Alt+/ gesture is convenient enough, but this gesture does not include the NVDA key, will it conflict with other software? Although I can't think of an exact example right now.

Thanks @cary-rowen for review
About the gesture, it is intended to resemble the context menu, so no need to NVDA key here, and when using it I feel it is near or like the context menu, for this I think it is appropriate. Moreover the user at any time, supposing this gesture has a role in some systems or regions, the user should learn to remove it through input gestures dialog, and assign for this purpose the gesture he wishes, so really I think the issue of the gesture should not be of big deal.
regards

Comment (cary-rowen) [reply addon/globalPlugins/openLinkWith/__init__.py:196]: OK, everything looks fine, we can merge it.

**Comment by ibrahim-s** [comment]
Hi @cary-rowen 
I have Enhanced the documentation, and the nvda.po for newly translatable messages.
If you find it now appropriate to merge, go on, and add the Chinease translation update , and then publish the stable version 2.9.
Cheers my friend

**Comment by cary-rowen** [comment]
Hi @ibrahim-s 
Great work!
I have updated the Simplified Chinese translation, I will merge it now.

Cheers

**Comment by ibrahim-s** [comment]
Great, @cary-rowen 
Thank you, and wish you all the good.

**Pull request merged**